tests/test_module_no_path_modified/module1/Snakefile

```diff
@@ -0,0 +1,7 @@
+rule a:
+    input:
+        path_modified("input.txt"),
```

Comment:
I don't really see why one would want this to be annotated manually. Can you explain?

Reply:
Described the problem a bit more in the description, so it is more visible.

Reply:
Ah, understood. I think this is covered semantically already by the `local` flag, see https://snakemake.readthedocs.io/en/stable/snakefiles/storage.html#local-input-output-files. 
However, it is not properly documented to also apply to module handling with prefixes. Can you check whether that works for you?

Reply:
No, the `local` flag does not work as the path is still modified.

Reply:
Ok, so reverted most changes and add a check for the `local` flag when modifying the path.